programmers/자릿수-더하기.js

```diff
@@ -0,0 +1,7 @@
+function solution(n) {
+    let result = 0;
+    
+    for (let i of String(n)) result += Number(i);
```

Comment:
```suggestion
    for (let i of String(n)) {
          result += Number(i);
    }
```
반복문이 실행되는 코드가 한 줄이어도 { }안에 넣어주시는 것이 좋습니다.